daemon/daemon_test.go

```diff
@@ -0,0 +1,77 @@
+package daemon
+
+import (
+	"testing"
+
+	"github.com/mesg-foundation/core/config"
+	"github.com/mesg-foundation/core/container"
+	"github.com/mesg-foundation/core/container/mocks"
+	"github.com/mesg-foundation/core/x/xnet"
+	"github.com/stretchr/testify/require"
+)
+
+func TestStart(t *testing.T) {
+	cfg, _ := config.Global()
+	c := &mocks.Container{}
+	d := NewContainerDaemon(cfg, c)
+
+	c.On("SharedNetworkID").Return("1", nil)
+	c.On("StartService", d.buildServiceOptions("1")).Return("1", nil)
+	d.Start()
+	c.AssertExpectations(t)
+}
+
+func TestStop(t *testing.T) {
+	cfg, _ := config.Global()
+	c := &mocks.Container{}
+	d := NewContainerDaemon(cfg, c)
+
+	c.On("StopService", []string{}).Return(nil)
+	d.Stop()
+	c.AssertExpectations(t)
+}
+
+func TestStatus(t *testing.T) {
+	cfg, _ := config.Global()
+	c := &mocks.Container{}
+	d := NewContainerDaemon(cfg, c)
+
+	c.On("Status", []string{}).Return(container.STOPPED, nil)
+	d.Status()
+	c.AssertExpectations(t)
+}
+
+func TestLogs(t *testing.T) {
+	cfg, _ := config.Global()
+	c := &mocks.Container{}
+	d := NewContainerDaemon(cfg, c)
+
+	c.On("ServiceLogs", []string{}).Return(nil, nil)
```

Comment:
need to return a real log stream to test the behavior

Reply:
My comment disapper, but you don't need to return writer, it's just a wrapper and tests are ok here